internal/builders/go/main.go

```diff
@@ -101,6 +101,7 @@ func runProvenanceGeneration(subject, digest, commands, envs, workingDir string)
 }
 
 func main() {
+	const defaultRekorAddr = "https://rekor.sigstore.dev"
```

Comment:
I was thinking to export this from the signing/sigstore package. something like `sigstore.DefaultRekorAddr`